crates/breez-sdk/lnurl/src/main.rs

```diff
@@ -0,0 +1,183 @@
+    pub domain: String,
+
+    /// Minimum amount that can be sent in a lnurl payment.
+    #[arg(long, default_value = "1000")]
+    pub min_sendable: u64,
+
+    #[arg(long, default_value = "4000000")]
+    pub max_sendable: u64,
+}
+
+type LnurlServer = routes::LnurlServer<Pool<ConnectionManager<SqliteConnection>>>;
+
+#[tokio::main]
+async fn main() -> Result<(), anyhow::Error> {
+    let args = Args::parse();
+    let config_file = std::fs::canonicalize(&args.config).ok();
+    let mut figment = Figment::new().merge(Serialized::defaults(args));
+    if let Some(config_file) = &config_file {
+        figment = figment.merge(Toml::file(config_file));
+    }
+
+    let args: Args = figment.merge(Env::prefixed("BREEZ_LNURL_")).extract()?;
+
+    tracing_subscriber::registry()
+        .with(EnvFilter::new(&args.log_level))
+        .with(tracing_subscriber::fmt::layer().with_writer(std::io::stdout))
+        .init();
+
+    if let Some(config_file) = &config_file {
+        info!(
+            "starting lnurl server with config file: {}",
+            config_file.display()
+        );
+    } else {
+        info!("starting lnurl server without config file");
+    }
+
+    // Create a connection manager for SQLite
+    let manager = ConnectionManager::<SqliteConnection>::new(&args.db_url);
+
+    // Create a connection pool
+    let pool = Pool::builder()
+        .build(manager)
+        .map_err(|e| anyhow!("failed to create connection pool: {:?}", e))?;
+
+    // Get a connection to run migrations
+    let mut db_connection = pool
+        .get()
+        .map_err(|e| anyhow!("failed to get connection from pool: {:?}", e))?;
+
+    if args.auto_migrate {
+        sqlite::run_migrations(&mut db_connection)?;
+    } else if sqlite::has_migrations(&mut db_connection)? {
+        return Err(anyhow::anyhow!(
+            "database has pending migrations, run with --auto-migrate to apply them, or apply them manually"
+        ));
+    }
+
+    let wallet = Arc::new(
+        spark_wallet::SparkWallet::connect(
+            SparkWalletConfig::default_config(args.network),
+            DefaultSigner::new(&[0; 32], args.network)?,
+        )
+        .await?,
+    );
+    let state = State {
+        db: Arc::new(pool),
+        wallet,
+        scheme: args.scheme,
+        domain: args.domain,
+        min_sendable: args.min_sendable,
+        max_sendable: args.max_sendable,
+    };
+
+    let server_router = Router::new()
+        .route("/lnurlpay/{pubkey}", post(LnurlServer::register))
+        .route("/lnurlpay/{pubkey}", delete(LnurlServer::unregister))
+        .route("/lnurlpay/{pubkey}/recover", delete(LnurlServer::recover))
+        .route(
+            "/.well-known/lnurlp/{identifier}",
+            get(LnurlServer::handle_lnurl_pay),
+        )
+        .route("/lnurlp/{identifier}", get(LnurlServer::handle_lnurl_pay))
+        .route(
+            "/lnurlp/{identifier}/invoice",
+            get(LnurlServer::handle_invoice),
+        )
```

Comment:
I think we are missing the verify route: https://github.com/breez/breez-lnurl/blob/main/lnurl/pay.go#L475